source/views/settings/icon.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+import React from 'react'
+import {ScrollView, Image, StyleSheet, Text} from 'react-native'
+import Icons from 'react-native-alternate-icons'
```

Comment:
This should be "import * as" (yes, that's not what their docs say, but their docs are wrong)